lib/posix-process/Makefile.uk

```diff
@@ -9,6 +9,9 @@ CXXINCLUDES-$(CONFIG_LIBPOSIX_PROCESS) += $(LIBPOSIX_PROCESS_COMMON_INCLUDES-y)
 
 LIBPOSIX_PROCESS_CFLAGS-$(CONFIG_LIBPOSIX_PROCESS_DEBUG) += -DUK_DEBUG
 
+LIBPOSIX_PROCESS_SRCS-$(CONFIG_LIBPOSIX_PROCESS_EXECVE) += $(LIBPOSIX_PROCESS_BASE)/execve.c
+LIBPOSIX_PROCESS_SRCS-$(CONFIG_LIBPOSIX_PROCESS_EXECVE) += $(LIBPOSIX_PROCESS_BASE)/arch/$(CONFIG_UK_ARCH)/execve.c|arch
```

Comment:
`|arch`? Maybe `|$(CONFIG_UK_ARCH)` or I am missing smth but I do not see `|arch` used anywhere else.

Reply:
That's just namespacing to avoid compile errors, as `execve.o` is already generated for `lib/posix-process/execve.c`. There are no semantics in `arch`, just a way to cause the build system generate `execve.arch.o`.